src/ontology/mondo-edit.obo

```diff
@@ -116017,7 +116017,7 @@ is_a: MONDO:0006600 {source="DOID:12311", source="EFO:1000770", source="linkedli
 [Term]
 id: MONDO:0006614
 name: subcorneal pustular dermatosis
-def: "Skin diseases characterized by local or general distributions of blisters. They are classified according to the site and mode of blister formation. Lesions can appear spontaneously or be precipitated by infection, trauma, or sunlight. Etiologies include immunologic and genetic factors. (From Scientific American Medicine, 1990)" [MESH:D012872]
+def: "Subcorneal pustular dermatosis is a rare, benign, chronic disease characterized by sterile pustular eruption, typically involving the flexural sites of the trunk and proximal extremities." [Orphanet:48377]
```

Comment:
See https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/
Do not replicate the name of the class in the definition